release notes/v0.47.0.md

```diff
@@ -75,12 +77,21 @@ _Format as `<number> <present_verb> <object>. <credit>`_:
 - [browser#1040](https://github.com/grafana/xk6-browser/pull/1040) fix the `context.clearCookies()` method to clear all cookies from the current browser context.
 - [browser#1031](https://github.com/grafana/xk6-browser/pull/1031) fix the `expires` field while adding cookies using the `context.addCookie()` method.
 - [browser#1039](https://github.com/grafana/xk6-browser/pull/1039) fix Goja conversions while adding and retrieving cookies.
+- [browser#1038](https://github.com/grafana/xk6-browser/pull/1038) Fix read/write race condition for edge case with remote browsers.
```

Comment:
Is this a race condition or a data race? Feels like the latter?

Reply:
Good spot, resolved in d09878a00a91225609d277fb4f56817bd67001d6